resources/definitions/fdmprinter.def.json

```
@@ -1649,6 +1649,19 @@
                     "limit_to_extruder": "top_bottom_extruder_nr",
                     "settable_per_mesh": true
                 },
+                "small_skin_width":
+                {
+                    "label": "Small Top/Bottom width",
```

Capitalize "width" for consistency.
```suggestion
                    "label": "Small Top/Bottom Width",
```